### ISSUE TITLE
Another potential false positive for Style/TrailingCommaInArguments

### ISSUE THREAD

**Issue by derekpovah** [rubocop/rubocop#10173]
In the example below, Rubocop expects commas between the `}` or `]` and the `)`, but I disagree.

I may be picking nits here since Ruby is a ninja and lets you omit curly braces when passing in `*args` to a method and it's generally the preferred way to handle an unknown number of method arguments. It becomes more relevant when the argument is a multi-line array, though.

Potentially related to #9935

--------

## Context

```yml
# .rubocop.yml
Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: consistent_comma
```

```ruby
# example.rb
klass.some_method(:foo, {
  bar: 'baz',
  qux: 'quux',
})

klass.another_method(:foo, [
  bar: 'baz',
  qux: 'quux',
])
```

## Expected behavior
```
$ rubocop example.rb --only Style/TrailingCommaInArguments

1 file inspected, no offenses detected
```

## Actual behavior
```
$ rubocop example.rb --only Style/TrailingCommaInArguments

C: [Correctable] Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.
        })
        ^
C: [Correctable] Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.
        ])
        ^

1 file inspected, 2 offenses detected, 2 offenses auto-correctable
```

## RuboCop version
```
$ [bundle exec] rubocop -V
1.22.1 (using Parser 3.0.2.0, rubocop-ast 1.12.0, running on ruby 2.7.3 x86_64-linux)
  - rubocop-rails 2.12.3
```
